extensions/mssql/src/main.ts

```diff
@@ -297,6 +336,21 @@ async function handleOpenNotebookTask(profile: azdata.IConnectionProfile): Promi
 	}
 }
 
+async function handleOpenClusterStatusNotebookTask(profile: azdata.IConnectionProfile): Promise<void> {
+	const notebookRelativePath = 'mssql/notebooks/TSG/cluster-status.ipynb';
+	const notebookFullPath = path.join(__dirname, '../../', notebookRelativePath);
```

Comment:
Does this logic work when the extension code is webpacked during full build? I am concerned relative lookup will fail. I believe going relative to the extension's root directory is the recommended way forward here. That's what we do for other resources in the notebook extension as far as I remember

Reply:
I updated it to get the path relative to the extension's root.